docs/guides/nginx-startup.md

```diff
@@ -280,6 +280,22 @@ The modules can also be built using an openresty archive, but in this case the b
 - replace `make modules` with `make && make install`
 - the two Brotli modules are built into the folder _/build/nginx-<version>/objs/_
 
+## Environment Variables
+
+NGINX environment variables need to be configured in the `cache.extraEnvVars` section of the PWA Helm Chart, e.g.
+
```

Comment:
link to chart repository would help here

Reply:
good point